worker/raft/rafttransport/worker_test.go

```diff
@@ -230,12 +231,28 @@ func (s *WorkerSuite) TestTransportWorkerStopped(c *gc.C) {
 
 func (s *WorkerSuite) TestTransportTimeout(c *gc.C) {
 	config := s.config
-	config.Timeout = time.Millisecond
+	config.Timeout = time.Nanosecond
```

Comment:
so I'm a little concern that any sort of timeout like this is a chance for our side to be paused, and the other side to actually get the work done before the timeout gets noticed.
(Most sleep sort of things are 'sleep at least this long', I certainly don't think they can guarantee delivery time.)

If it was matched times "we set a deadline of 1ms, and on the other side we force a 10ms sleep", then I would be happier about it.